example-app/SantokuApp/.script/print-eslint-rules.js

```diff
@@ -0,0 +1,193 @@
+    name: '@typescript-eslint',
+    prefix: '@typescript-eslint',
+    url: 'https://typescript-eslint.io/',
+    rulesRootPath: 'rules',
+    hasPagePerRule: true,
+  },
+  {
+    name: 'eslint-plugin-import',
+    prefix: 'import',
+    url: 'https://github.com/import-js/eslint-plugin-import',
+    rulesRootPath: 'blob/main/docs/rules',
+    extension: '.md',
+    hasPagePerRule: true,
+  },
+  {
+    name: 'eslint-plugin-eslint-comments',
+    prefix: 'eslint-comments',
+    url: 'https://mysticatea.github.io/eslint-plugin-eslint-comments',
+    rulesRootPath: 'rules',
+    extension: '.html',
+    hasPagePerRule: true,
+  },
+  {
+    name: 'eslint-plugin-react-hooks',
+    prefix: 'react-hooks',
+    url: 'https://github.com/facebook/react/tree/main/packages/eslint-plugin-react-hooks',
+  },
+  {
+    name: 'eslint-plugin-prettier',
+    prefix: 'prettier',
+    url: 'https://github.com/prettier/eslint-plugin-prettier',
+  },
+  {
+    name: 'eslint-plugin-react',
+    prefix: 'react',
+    url: 'https://github.com/jsx-eslint/eslint-plugin-react',
+    rulesRootPath: 'blob/master/docs/rules',
+    extension: '.md',
+    hasPagePerRule: true,
+  },
+  {
+    name: 'eslint-plugin-strict-dependencies',
+    prefix: 'strict-dependencies',
+    url: 'https://github.com/knowledge-work/eslint-plugin-strict-dependencies',
+  },
+  {
+    name: 'eslint-plugin-deprecation',
+    prefix: 'deprecation',
+    url: 'https://github.com/gund/eslint-plugin-deprecation',
+  },
+  {
+    name: 'eslint-plugin-node',
+    prefix: 'node',
+    url: 'https://github.com/mysticatea/eslint-plugin-node',
+    rulesRootPath: 'blob/master/docs/rules',
+    extension: '.md',
+    hasPagePerRule: true,
+  },
+  {
+    name: 'eslint-plugin-jest',
+    prefix: 'jest',
+    url: 'https://github.com/jest-community/eslint-plugin-jest',
+    rulesRootPath: 'blob/main/docs/rules',
+    extension: '.md',
+    hasPagePerRule: true,
+  },
+];
+
+const isDisabled = level => {
+  // https://eslint.org/docs/latest/use/configure/configuration-files-new#rule-severities
+  if (typeof level === 'number' && level === 0) {
+    return true;
+  }
+  return level === 'off';
+};
+
+/**
+ * 以下の形式でルールページを作成します。
+ * [プラグインURL]/[ルールページのルートパス]/[プレフィックスを除いたルール名][拡張子]
+ */
+const generateRuleUrl = ({prefix, url, rulesRootPath, extension = '', hasPagePerRule, rule}) => {
+  if (hasPagePerRule) {
+    const rulePagePath = rule.replace(prefix, '') + extension;
+    return urlJoin(url, rulesRootPath, rulePagePath);
+  }
+  return url;
+};
+
+const tableHeader = `|ルール|レベル|\n|:--|:--|`;
+const transformToTableRow = ({rule, ruleUrl, level}) => `|[${rule}](${ruleUrl})|${level}|`;
+const transformToH3 = ({pluginName, pluginUrl}) => `### [${pluginName}](${pluginUrl})`;
+
+/**
+ * ESLintのルール一覧をマークダウン形式で標準出力します。
+ * 出力形式は以下とします。
+ * H3: プラグイン名
+ * Table: ルール名とレベルの一覧
+ *
+ * e.g.
+ * ### [@typescript-eslint](https://github.com/typescript-eslint/typescript-eslint/tree/master/packages/eslint-plugin)
+ * |ルール|レベル|
```

Comment:
改行が追加されていたので、 `e.g.` にも追加

``return `${h3}\n\n${tableHeader}\n${rows.join('\n')}`;``

```suggestion
 * ### [@typescript-eslint](https://github.com/typescript-eslint/typescript-eslint/tree/master/packages/eslint-plugin)
 
 * |ルール|レベル|
```